views/js/controller/viewResult.js

```diff
@@ -65,13 +83,13 @@ define([
             });
 
             $('.preview', $container).on("click", function (e) {
```

Comment:
arrow fn and single quotation?

Reply:
I've updated the quote but let the function since the lexical scope is used within it